components/DogList/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import DogCard from "/components/DogCard";
+
+const DogList = () => {
```

Comment:
Also here it's not the right syntax to create a component like we learned it in our sessions.

Your `DogList` component should be written more like the following, or? In this component you accept the prop `dogs` (it's the data from your mock array in the `_app.js`) and map through the `dogs` array to render one `DogCard` per object in the `dogs` array and pass down the (singular) prop `dog` to the `DogCard` component. 

```
import DogCard from "/components/DogCard";

export default function DogList({ dogs }) {
  return (
    <ul>
      {dogs.map((dog) => (
        <DogCard key={dog.name} dog={dog} />
      ))}
    </ul>
  );
}


```